src/time-picker/TimePicker.js

```diff
@@ -0,0 +1,211 @@
+import React from 'react';
+import { props, t, skinnable } from '../utils';
+import cx from 'classnames';
+import Dropdown from '../dropdown/Dropdown';
+import range from 'lodash/utility/range';
+import flatten from 'lodash/array/flatten';
+
+/*
+  Options is defined here cause we need to not change the ref.
+  This happend because of a very wired behaviour of react-select v0.6.12
```

Comment:
typos ^

however, a comment like `hardcoded empty options. We are generating options dynamically` would be more useful I think